index.html

```diff
@@ -8286,11 +8286,12 @@ <h3>Ticks</h3>
  content observable effects that must be consistent across
  implementations. To accommodate this, the specification requires
  that <a>remote ends</a> <dfn>perform implementation-specific action
- dispatch steps</dfn> on a <a>browsing context</a> <var>context</var>,
- and a <var>list of events</var> and their properties. These steps
- must be equivalent to performing the given input device manipulations
- on <var>context</var>, such that trusted events corresponding to the
- entries in <var>list of events</var>are dispatched.
+ dispatch steps</dfn> on a <var>context</var>, and a <var>list of events</var>
+ and their properties. These steps must be equivalent to performing the given
+ input device manipulations on the
+ [=browsing context/top-level browsing context=] of the <var>context</var>,
+ such that trusted events corresponding to the entries in
+ <var>list of events</var> are dispatched.
```

Comment:
We can "hand-wave" hear like "These steps must be equivalent to user trying to perform the given input device manipulations on context through the top-level browsing context."

Reply:
https://github.com/w3c/webdriver/pull/1850

Reply:
@OrKoN @whimboo we can keep specification implementation-specific, just clarify the events are dispatched via top-level browsing context. I'm not sure if this approach would work with the events intercepted by the top-level frame.